### PR TITLE
Fish : LDFLAGS required for backtraces support.

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

File: app-shells/fish/fish-2.4.0.recipe [hunk 1]
```diff
@@ -59,8 +59,14 @@ GLOBAL_WRITABLE_FILES="
 BUILD()
 {
 	autoreconf -vfi
-	CFLAGS=-D_BSD_SOURCE LDFLAGS="-lnetwork" runConfigure ./configure \
+
+	export CFLAGS="-D_BSD_SOURCE"
+	export CXXFLAGS=$CFLAGS
+	export LIBS="-lnetwork -lexecinfo"
```
Comment (korli) [inline app-shells/fish/fish-2.4.0.recipe]: You should amend configure.in with something like:
AC_SEARCH_LIBS(socket, socket network)
AC_SEARCH_LIBS(backtrace, execinfo)
and remove LDFLAGS from the recipe.

Comment (waddlesplash) [inline app-shells/fish/fish-2.4.0.recipe]: Why is that needed, exactly? This method works properly.

Comment (miqlas) [inline app-shells/fish/fish-2.4.0.recipe]: AC_SEARCH for socket waiting for merge in upstream : https://github.com/fish-shell/fish-shell/pull/3675
AC_Search for execinfo already in the latest git version.
So i just don't want to invest too much work in this commit right now.

Comment (waddlesplash) [inline app-shells/fish/fish-2.4.0.recipe]: Fine by me.
